release-notes/release-notes58.md

```diff
@@ -101,6 +101,32 @@ The {{site.productname}} 5.8 release includes an accompanying release of the **E
 
 For information on the Export plugin, see: [Export plugin]({{site.baseurl}}/plugins/premium/export/).
 
+### PowerPaste 5.5.0
+
+The {{site.productname}} 5.8 release includes an accompanying release of the **PowerPaste** premium plugin.
+
+This release adds support for copying and pasting content from Google Docs, making it possible to copy and retain styles from Google Docs content. By default, a dialog will now appear asking users if the content should be cleaned or merged when pasting Google Docs content.
```

Comment:
```suggestion
This release adds support for copying and pasting content from Google Docs, making it possible to copy content and retain styles from Google Docs. By default, a dialog will appear asking users if the content should be cleaned or merged when pasting Google Docs content.
```
With `content` there, it reads like we're copying the styles, not the content and styles